network-api/networkapi/wagtailpages/templatetags/homepage_tags.py

```diff
@@ -32,17 +32,16 @@ def homepage_image_with_class(context, path, classname):
 
 
 @register.simple_tag
-def get_blog_page_authors(blog_page):
+def get_page_authors(parent_page):
```

Comment:
It appears that this function already existed to return the authors of a blog page, I have updated it to accept both blog pages and publication pages, that way we can continue to use the single "author-images" template by calling this tag in both the blog page template and publication page template